skypilot-config.yaml

```diff
@@ -398,5 +401,9 @@ setup: |
   git clean -fd
   # Install astral-uv
   sudo snap install --classic astral-uv
+  # Install tmux
+  apt install tmux -y
+  # install awscli
+  uv add awscli
```

Comment:
We should probably add this as a dependency of openpipe-art (I realize I should have done that).

Reply:
I think it is already added to ART, so we can remove this line from the script.